sklearn/linear_model/ridge.py

```diff
@@ -847,8 +847,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : Returns self.
         """
+        alphas = np.asarray(self.alphas)
```

Comment:
I would go with the least number of lines to modify and go with the tuple.
Though, the doc says that it accepts (only) numpy array. http://scikit-learn.org/dev/modules/generated/sklearn.linear_model.RidgeClassifierCV.html#sklearn.linear_model.RidgeClassifierCV  From a user perpsective, I would expect array-like for alphas.

In that case, I would add a regression support and add support for array-like.